code/controllers/subsystem/minor_mapping.dm

```diff
@@ -84,5 +97,23 @@ SUBSYSTEM_DEF(minor_mapping)
 
 	return shuffle(suitable)
 
+/// Find station maintenance turfs that are not blocked
+/proc/find_unblocked_maint_turfs()
```

Comment:
this seems like it's going to cost quite a bit on perf.

add `AUTO_PROFILE` to your config, start an init, then check the server profiler, find this proc, and show me a picture of the numbers

i feel like this should be using our existing spawners instead

Reply:
![image](https://github.com/tgstation/tgstation/assets/40921881/4855d669-7b78-4d8d-aed4-6410fc4bb1c5)


I already didn't like this solution (it can spawn it right next to maintenance entrances) so im going to look into creating a rare maint structure spawn point and sprinkling that around the maps, maybe add the cult pant thing to it too. That way I can control it not spawning right next to an airlock or something.

Reply:
wait one sec that was on runtime station

Reply:
Delta Station:

![image](https://github.com/tgstation/tgstation/assets/40921881/72c317da-c6fa-44cb-b8de-c2a6cf2e9448)